src/transform/convert/generate-static-fields.spec.js

```diff
@@ -27,10 +37,23 @@ function callback({
 }) {
   const inputData = new MarcRecord(getFixture('input.json'));
   const expectedResults = getFixture('output.json');
-  const result = handleProcess(functionToUse, inputData) // eslint-disable-line
+  const result = handleProcess(functionToUse, inputData); // eslint-disable-line
 
-  expect(result).to.eql(expectedResults);
+  if (result) {
```

Comment:
Each tests should probably have some kind of result.

Reply:
In normal run error cases deviate record handling to different end. so there is no error result that can returns